src/print.cc

```diff
@@ -428,18 +427,18 @@ std::string tile_row_string(
 template <typename scalar_t>
 void print_work(
     const char* label,
-    slate::BaseMatrix<scalar_t>& A,
+    BaseMatrix<scalar_t>& A,
     int64_t klt,
     int64_t kut,
-    slate::Options const& opts_)
+    Options const& opts_)
 {
     using real_t = blas::real_type<scalar_t>;
-    slate::Options opts(opts_);
+    Options opts( opts_ );
```

Comment:
Side remark, usually in the source files, when we declare local options, we do
```
Options local_opts = opts;
```
We should be consistent on this.